packages/cms/pages/auth/register/register.tsx

```diff
@@ -12,12 +12,30 @@ import { EuiFlexGroup } from '@tensei/eui/lib/components/flex'
 import { EuiFieldText } from '@tensei/eui/lib/components/form/field_text'
 import { EuiFieldPassword } from '@tensei/eui/lib/components/form/field_password'
 import { EuiFlexItem } from '@tensei/eui/lib/components/flex/flex_item'
+import { useState } from 'react'
 
 const H3 = styled.h3`
   text-align: center;
 `
 
 export const Register: React.FunctionComponent = () => {
+  const [user, setUser] = useState({
+    firstname: '',
+    lastname: '',
+    email: '',
+    password: ''
+  })
+
+  const submitRegistration = async (e: any) => {
+    try {
+      e.preventDefault()
+      let resp = await window.Tensei.client.post('users', user)
+      console.log(resp)
+    } catch (error) {
+      console.log(error.message)
```

Comment:
Let's set the validation errors to state and pass them to the components if there are any. Also, let's call `window.location.href = window.Tensei.getPath('')`. This will redirect the user to the dashboard, and they'd already be logged in. 

The backend endpoints will set session cookies on the browser, and after the page refresh / redirect, the user will be logged in.

Reply:
okay, thanks.